VirtualEnvWrapper.psm1

```diff
@@ -98,7 +98,7 @@ function Invoke-CreatePyEnv($Command, $Name) {
     $NewEnv = Join-Path $WORKON_HOME $Name
     Write-Host "Creating virtual env... "
     
-    Invoke-Expression "$Command $NewEnv"
+    Invoke-Expression "$Command '$NewEnv'"
```

Comment:
This change is still missing on master. If that is added spaces in folder names work for Python3 at least.